modules/rosa-adding-taints-cli.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+// * osd_cluster_admin/osd_nodes/osd-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-adding-taints-cli{context}"]
+= Adding taints to a machine pool using ROSA CLI
+
+You can add taints to a machine pool for your Red Hat OpenShift Service on AWS (ROSA) cluster by using the ROSA CLI.
+
+.Prerequisites
+
+ifdef::openshift-rosa[]
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+* You created an {product-title} cluster.
+endif::[]
+* You have an existing machine pool that does not contain any taints and contains at least two instances.
+
+.Procedure
+
+ifdef::openshift-rosa[]
+. List the machine pools in the cluster:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONES    SPOT INSTANCES
+Default      No           2         m5.xlarge                          us-east-1a            N/A
+db-nodes-mp  No           2         m5.xlarge                          us-east-1a            No
+----
+
+. Add or update the taints for a machine pool:
+
+* To add or update taints for a machine pool that does not use autoscaling, run the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> \
+                        --replicas=<replica_count> \ <1>
+                        --taints=<key>=<value>:<effect>,<key>=<value>:<effect> \ <2>
+                        <machine_pool_id>
+----
+<1> For machine pools that do not use autoscaling, you must provide a replica count when adding taints. If you do not specify the `--replicas` argument, you are prompted for a replica count before the command completes. If you deployed {product-title} (ROSA) using a single availability zone, the replica count defines the number of compute nodes to provision to the machine pool for the zone. If you deployed your cluster using multiple availability zones, the count defines the total number of compute nodes in the machine pool across all zones and must be a multiple of 3.
+<2> Replace `<key>=<value>:<effect>,<key>=<value>:<effect>` with a key, value, and effect for each taint, for example `--taints=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.This list overwrites any modifications made to node taints on an ongoing basis.
```

Comment:
Should key, value, and effect be literals?
Also you need a space between `NoExecute`. and This